src/plugins/index_pattern_management/public/components/index_pattern_table/index_pattern_table.tsx

```diff
@@ -47,7 +47,7 @@ const search = {
 };
 
 const title = i18n.translate('indexPatternManagement.dataViewTable.title', {
-  defaultMessage: 'Data views',
+  defaultMessage: 'Data Views',
```

Comment:
Hmm, I was under the impression we almost always preferred sentence case, even in titles: https://elastic.github.io/eui/#/guidelines/writing

Reply:
@gchaps corrected me

Reply:
We do prefer sentence case, as the guidelines suggest.  I capitalized Data View here because it is a feature name and also to match the title caps in the left nav bar in Stack Management.